src/coreclr/src/vm/tailcallhelp.cpp

```diff
@@ -163,9 +157,9 @@ void TailCallHelp::CreateTailCallHelperStubs(
 
 void TailCallHelp::LayOutArgBuffer(
     MetaSig& callSiteSig, MethodDesc* calleeMD,
-    bool storeTarget, bool thisArgByRef, ArgBufferLayout* layout)
+    bool storeTarget, bool thisArgByRef, bool hasInstArg, ArgBufferLayout* layout)
 {
-    unsigned int offs = 0;
+    unsigned int offs = offsetof(TailCallArgBuffer, Args);
```

Comment:
The size is assigned from this offset (at the end of the function), so the size is a bit overapproximated now.

Reply:
Good catch. Fixed.